extensions/amp-analytics/analytics-vars.md

```diff
@@ -74,6 +74,12 @@ Provides the canonical URL of the current document.
 
 Example value: `http%3A%2F%2Fexample.com%3A8000%2Fanalytics.html`
 
+### documentCharset
```

Comment:
yes, the documentCharset was in the wrong category. I moved it to the correct one. I can send out a separate PR if that is preferred.